playground/src/main/scala/sttp/tapir/example/zio/LayerEndpoint.scala

```diff
@@ -0,0 +1,28 @@
+package sttp.tapir.examples
```

Comment:
the examples project is where examples should live, playground is just for quick experimentation :)

Reply:
Got your message. So what to do  ?

Reply:
I'd just leave the example in the `example` module and remove from `playground` :)

Reply:
Done